internal/log/log.go

```diff
@@ -0,0 +1,37 @@
+package log
+
+import (
+	"fmt"
+	"github.com/logrusorgru/aurora"
+)
+
+// Au Aurora instance used for colors
+var Au aurora.Aurora
+
+// Print print a message with formatting
+func Print(part string, parts ...interface{}) {
+	hoverPrint()
+	fmt.Println(fmt.Sprintf(part, parts...))
+}
+
+// Fatal print a error with formatting (red)
+func Fatal(part string, parts ...interface{}) {
```

Comment:
The name `Fatal` is misleading. In other log packages, Fatal also calls `os.Exit`. Perhaps change this to `Error`.